app/sqlitehelper.py

```diff
@@ -0,0 +1,31 @@
+import sqlite3
+from sqlite3 import Connection, Error
+
+DEFAULT_DATABASE = "sqlitedatabase.db"
+
+class SQLiteHelper:
```

Comment:
I don't think SQLiteHelper is a good class name. Perhaps something like SQLiteDatabase, Database (is the sqlite part even important?), DataManager, ConnectionManager, DatabaseManager, etc.

I should be able to look at the class name and have a rough understanding of what the class does.

Reply:
I totally agree. I have changed it in the latest commit